Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM alpine
-MAINTAINER Kolide Developers <engineering@kolide.co>
-
-RUN apk --update add ca-certificates
+FROM gcr.io/distroless/base-debian10:nonroot
+LABEL author Kolide Developers <engineering@kolide.co>
```

Comment:
Let's use
```
LABEL maintainer="engineering@kolide.co"
```
as suggested in https://docs.docker.com/engine/reference/builder/#maintainer-deprecated